observatorium/grafana.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: integreatly.org/v1alpha1
+kind: Grafana
+metadata:
+    annotations:
+        argocd.argoproj.io/sync-wave: "1"
+    name: grafana
+spec:
+    baseImage: quay.io/app-sre/grafana:7.1.1
+    config:
+        security:
+            admin_user: "root"
```

Comment:
can we move this to a secret? https://github.com/AICoE/internal-data-hub/pull/17

Reply:
It doesn't really matter since the login page is disabled you cannot log in with any credentials

Reply:
This was copied from https://github.com/operate-first/apps/blob/master/odh/base/monitoring/grafana.yaml